packages/ibm-products/src/components/Checklist/ChecklistChart.js

```diff
@@ -50,18 +50,14 @@ const defaults = {
  * TODO: A description of the component.
  */
 export let ChecklistChart = React.forwardRef(
-  (
-    { className, ariaLabelledById, value, theme = defaults.theme, ...rest },
-    ref
-  ) => {
+  ({ className, value, theme = defaults.theme, ...rest }, ref) => {
```

Comment:
Removed `ariaLabelledById`. Using built-in `aria-labelledby` via `...rest` instead.